prow/scripts/cluster-integration/compass-gke-benchmark.sh

```diff
@@ -151,7 +151,7 @@ function installHelm() {
 }
 
 function installKyma() {
-  KYMA_CLI_VERSION="2.0.4"
+  KYMA_CLI_VERSION="2.1.3."
```

Comment:
Version number should be `2.1.3`

Reply:
Also potentially missed override changes in `compass-gke-kyma-overrides.tpl.yaml`

Reply:
What this job is used for? I think it's a leftover?

Reply:
Currently in the file mentioned by Ivan, the istio overrides are `istio-configuration:`. However, with the new kyma (and as they are adapted in the config repo), the istio overrides should be: `istio:`